.gitignore

```diff
@@ -1,7 +1,9 @@
 vendor
+/node/.vendor-new/*
```

Comment:
I think this is from an inconsistent state in dep, so it shouldn't be in the .gitignore...